locales/en.json

```diff
@@ -220,7 +220,6 @@
     "log_help_to_get_log": "To view the log of the operation '{desc}', use the command 'yunohost log display {name}'",
     "log_link_to_failed_log": "The operation '{desc}' has failed ! To get help, please <a href=\"#/tools/logs/{name}\">provide the full log of this operation by clicking here</a>",
     "log_help_to_get_failed_log": "The operation '{desc}' has failed ! To get help, please share the full log of this operation using the command 'yunohost log display {name} --share'",
-    "log_category_404": "The log category '{category}' does not exist",
```

Comment:
Why this change? I don't see how it is related to this PR :o

Reply:
That it's true. I've taken the liberty to remove this line cause it's a duplicate of the line 218. But I can revert that.

Reply:
It's more that we shouldn't mix different things in the same PR and do atomic commits, that makes things way easier to handle :)

Reply:
I understand that, you what I revert this line and submit another PR with this line?

Reply:
No, that's for next time :)